ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactZIndexedViewGroup.java

```diff
@@ -12,4 +12,10 @@
    * @return The child view index considering z-index
    */
   int getZIndexMappedChildIndex(int index);
+
+  /**
+   * Redraw the view based on updated z-index. This should be called after updating one of it's child
+   * z-index.
+   */
+  void updateZIndex();
```

Comment:
Yea I had a hard time figuring out a name. `updateViewParentZIndex` would not work in this case because it doesn't update the parent it redraws the view based on the updated z-index of its children.